### ISSUE TITLE
Remove features related to legislation

### ISSUE THREAD

**Issue by brylie** [CiviWiki/OpenCiviWiki#857]
The legislation feature is currently U.S.-centric, making this project less relevant in other countries. We rely on third-party data sources for information on legislation, which adds a maintenance burden that this project cannot bear. Furthermore, we cannot support adding legislation for other political forms.

Remove code related to legislation as part of simplifying our data model, removing third-party integrations, and making the project useful in an international context.

## Related code
Remove related code from the following sections.

:warning: **Note** Remember to run the [`makemigrations` command](https://docs.djangoproject.com/en/3.2/ref/django-admin/#django-admin-makemigrations) after changing the project models.

### Model(s)
https://github.com/CiviWiki/OpenCiviWiki/blob/f152ad5986ebfd25ff60ad5ca7c34563818f1e17/project/api/models/bill.py

### Accounts model
https://github.com/CiviWiki/OpenCiviWiki/blob/f152ad5986ebfd25ff60ad5ca7c34563818f1e17/project/api/models/account.py#L47

https://github.com/CiviWiki/OpenCiviWiki/blob/f152ad5986ebfd25ff60ad5ca7c34563818f1e17/project/api/models/account.py#L285-L322

### CiviWiki models
https://github.com/CiviWiki/OpenCiviWiki/blob/f152ad5986ebfd25ff60ad5ca7c34563818f1e17/project/api/models/rationale.py#L9

https://github.com/CiviWiki/OpenCiviWiki/blob/f152ad5986ebfd25ff60ad5ca7c34563818f1e17/project/api/models/rationale.py#L3

### Templates
Remove the following files.

https://github.com/CiviWiki/OpenCiviWiki/blob/ff74546f8fdc277c9bc5705f3df0c807c1130c62/project/webapp/templates/partials/account/tabs/my_bills.html

### Data integration code
Remove the following
https://github.com/CiviWiki/OpenCiviWiki/blob/f152ad5986ebfd25ff60ad5ca7c34563818f1e17/project/api/tests/propublica_responses.py

https://github.com/CiviWiki/OpenCiviWiki/blob/f152ad5986ebfd25ff60ad5ca7c34563818f1e17/project/api/propublica.py

### CiviWiki API/design files
https://github.com/CiviWiki/OpenCiviWiki/blob/ff74546f8fdc277c9bc5705f3df0c807c1130c62/docs/design/bill_api.json

https://github.com/CiviWiki/OpenCiviWiki/blob/f152ad5986ebfd25ff60ad5ca7c34563818f1e17/project/api/views/bill.py

### Votes
https://github.com/CiviWiki/OpenCiviWiki/blob/f152ad5986ebfd25ff60ad5ca7c34563818f1e17/project/api/models/vote.py

https://github.com/CiviWiki/OpenCiviWiki/blob/f152ad5986ebfd25ff60ad5ca7c34563818f1e17/project/api/tasks.py#L74-L80

### Management command
https://github.com/CiviWiki/OpenCiviWiki/blob/f152ad5986ebfd25ff60ad5ca7c34563818f1e17/project/api/management/commands/gather_votes_data.py

Delete the `api/management/command/` directory, since it only contains one file.

**Comment by Swanand01** [comment]
Hello @brylie, I am new to the open source community. I would like to take up this issue. Please brief me on the issue.

**Comment by brylie** [comment]
Hi @Swanand01, this issue involves removing some existing code. I can direct you towards the code to remove if that would help.

**Comment by Swanand01** [comment]
Yes @brylie that would definitely help. Thanks

**Comment by brylie** [comment]
@Swanand01 I am available for the next hour in a [CiviWiki office hours hangout](https://codebuddies.org/hangout/qmdQirv9AhdNYkyGo) on Codebuddies.org.

**Comment by brylie** [comment]
@tberm please also be aware of this issue as it is closely related to #830

**Comment by brylie** [comment]
@Swanand01 please consider collaborating with @tberm on this issue, so that your work aligns with one another.

**Comment by brylie** [comment]
@Swanand01 and @tberm, I have updated the issue description with references to related code.

**Comment by Swanand01** [comment]
Hi @brylie, I have performed all the above changes, but I am stuck on the makemigrations part.
```
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Requested setting TEMPLATES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
```

This is the error I got when I ran django-admin makemigrations from the project directory.
Thanks.

**Comment by brylie** [comment]
@Swanand01 please push your code to GitHub and open a pull request. Add the following text in the first line of your pull request description:

```
Closes #857
```

I will take a look at your code and try to reproduce or fix the issue.
